### PR TITLE
Subheadings font-weight

### PR DESCRIPTION
**Pull request by radiocontrolled**
Resolves https://github.com/bbc/simorgh/issues/710

Gives the subheading a font-weight of 400 ([normal](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Values)). 

**Screenshots**

latest branch:
screen shot 2018-09-24 at 14 33 43

subheadings-font-weight branch:
screen shot 2018-09-24 at 14 33 51

- [N/A] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by bcmn — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]
In the future we may want to move this value into a constant which could allow us to have `FW_LIGHT`, `FW_NORMAL` and `FW_BOLD`. This then gives us the flexibility to easily override bold or light if they aren't valid for a given script/typeface, which is likely to be needed for WS sites

**Comment by jamesbrumpton** [comment]
LGTM. This can be merged once the merge conflict has been dealt with.

**Pull request merged**
